includes/common.php

```diff
@@ -1888,3 +1888,11 @@ function array_by_column($array, $column)
 {
     return array_combine(array_column($array, $column), $array);
 }
+
+/**
+ * Dump and die
+ */
+function dd() {
```

Comment:
Wrap this in a function exists check for when we switch to Laravel?

Reply:
@laf sorry, this was for someone to test.  Not ready for review fully yet :)